core/src/main/resources/reference.conf

```diff
@@ -100,6 +100,11 @@ akka.persistence.r2dbc {
 
   connection-factory {
     driver = "postgres"
+
+    # the connection can be configured with a url, eg: "r2dbc:postgresql://<host>:5432/<database>"
+    url = null
```

Comment:
I don't mind adding this, but curious about if there is any background to why we need an alternative way?

Reply:
True, actually I took this path because I want to add `DB_CLOSE_DELAY=-1` to the URL. Which is a source of silly mistakes when using h2 in tests. 

We can pass arbitrary options though, but I thought that it would be better to have one simple way of configured a in-mem db instead of enriching the settings with extra things.